beancount_import/source/ofx.py

```diff
@@ -797,6 +833,10 @@ def get_security(unique_id: str) -> Optional[str]:
                 return None
             sec = securities_map[unique_id]
             ticker = sec.ticker
+            # Treasury bill and bond start with 912
+            if ticker.startswith("912"):
```

Comment:
FYI, this will crash if the ticker is `None`, rather than printing a helpful error message as it used to.